src/app/page.tsx

```diff
@@ -2,15 +2,13 @@ import Image from "next/image";
 
 export default function Home() {
   return (
-    <main className="bg-[#0A142F]">
+    <main className="sm:grid bg-[#0A142F]">
       <div className="relative min-h-screen">
-        <Image
-          src="/page2.svg"
-          alt="Next.js Logo"
-          layout="fill"
-          objectFit="contain"
-          priority
-        />
+        <div className=" m-8 grid min-h-[1000px] bg-red-700 sm:grid-cols-2">
+          <div className=" m-6 gird min-h-[300px] bg-black"></div>
+          <div className=" m-6 gird min-h-[300px] bg-yellow-400"></div>
+          <div className=" m-6 gird min-h-[100px] min-w-[2000px] bg-green-500"></div>
+        </div>
```

Comment:
make sure the design fits on smaller screens(try to make the width of the page smaller than 2000px) and try to make the height smaller than 1000px. The proportions should look the same. Try using flex, and flex-grow containers instead of grid